pkg/querier/queryrange/roundtrip.go

```diff
@@ -321,13 +320,45 @@ func (r roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 
 		return r.indexStats.RoundTrip(req)
 	case SeriesVolumeOp:
+		queryReq, err := LokiCodec.DecodeRequest(req.Context(), req, nil)
```

Comment:
We talked about putting this here but I'm concerned that nothing else is using the codec in `RoundTrip`. Would it be better to copy the instant and range examples and have separate parse functions? wdyt?